ingest/ingest_pipeline.py

```diff
@@ -576,29 +576,35 @@ def main() -> None:
     Returns:
         None
     """
+    parsed_args = create_parser().parse_args()
+    validate_arguments(parsed_args)
+    arguments = vars(parsed_args)
     status = 0
+    status_cell_metadata = 0
+    ingest = None
+
+    # Initialize global variables for current ingest job
+    config.init(
```

Comment:
local testing shows this erroring out:
```
Traceback (most recent call last):  
  File "ingest_pipeline.py", line 614, in <module>  
    main()  
  File "ingest_pipeline.py", line 608, in main  
    MetricsService.log(config.get_parent_event_name(), config.get_metric_properties())  
  File "/Users/jlchang/Documents/GitHub/scp-ingest-pipeline/ingest/config.py", line 28, in get_parent_event_name  
    return __event_name  
NameError: name '__event_name' is not defined
```
Moving `try:` down one line so the IngestPipeline object is instantiated should resolve this error.